src/crawlee/playwright_crawler/_playwright_crawler.py

```diff
@@ -80,20 +82,35 @@ def __init__(
             browser_pool: A `BrowserPool` instance to be used for launching the browsers and getting pages.
             browser_type: The type of browser to launch ('chromium', 'firefox', or 'webkit').
                 This option should not be used if `browser_pool` is provided.
+            browser_options: Keyword arguments to pass to the browser launch method.
+                This option should not be used if `browser_pool` is provided.
+            page_options: Keyword arguments to pass to the new page method.
+                This option should not be used if `browser_pool` is provided.
             headless: Whether to run the browser in headless mode.
                 This option should not be used if `browser_pool` is provided.
             kwargs: Additional keyword arguments to pass to the underlying `BasicCrawler`.
         """
         if browser_pool:
             # Raise an exception if browser_pool is provided together with headless or browser_type arguments.
-            if headless is not None or browser_type is not None:
+            if (
```

Comment:
With many conditions like this you could consider using **if any**(headless, browser_type, browser_options, page_options)

Reply:
I guess that:
 if any(param is not None for param in (headless, browser_type, browser_options, page_options))
insetad of 
if any(headless, browser_type, browser_options, page_options) 
is there because of mypy want's explisit is not none